ticoniq/blog
# Add device

## Devise Integration and User Authentication

This pull request implements user authentication and registration functionality using the Devise gem. It includes the following features and changes:

### Features Implemented

1. **Devise Integration**: Devise has been installed and set up in the application to handle user authentication.

2. **User Registration**: Users can now register by providing their email and password.

3. **Login**: Users can log in using their registered email and password combination.

4. **Password Hashing**: User passwords are securely hashed and stored in the database.

5. **Email Confirmation**: A confirmation email is sent to the user's email address upon registration.

6. **Password Reset**: Users can request a password reset if they forget their password.

7. **Styled Views**: The registration and login views have been modified and styled to match the wireframes provided in the Sneak Peek.

### How to Test

1. Navigate to the registration page and sign up with a new user account.

2. Confirm your email address by clicking the confirmation link sent to your
